src/net/sourceforge/kolmafia/persistence/MonsterDatabase.java

```diff
@@ -66,7 +66,17 @@ public enum Element {
         "supercold",
         "ice.gif",
         "#ADD8E6", // lightblue
-        "is Supercold");
+        "is Supercold."),
+    BADSPELLING(
+        "bad spelling",
+        "cookbook.gif",
+        "#800080", // purple
+        "is Bad Spelling. Bad Spelling is weak against dictionaries."),
```

Comment:
If the monsters with Bad Spelling attacks are, actually, vulnerable against dictionaries, does that mean they are also ED: "bad spelling" - or actually "E: bad spelling"?

That might be a can of worms; Monster Manuel claims they all have no elemental alignment.
I don't think we need to go there. :)

Reply:
As-is, it was a bit of a joke with some in-game relation: monsters that attack with bad spelling come from The Valley of Rof L'm Fao, and monsters from the valley are vulnerable to a dictionary. Outside the game, a dictionary might fix bad spelling ;)

I don't actually know what the point of bad spelling attacks are, because you can't resist "bad spelling". The monsters with bad spelling elemental attacks are vulnerable to dictionaries, so possibly this was the game's way of indicating that? This was far before manuel. Bad ASCII Art doesn't have any "bad spelling" attacks but is still vulnerable to the dictionary, but again that one came far after the others.

There are various elements that don't show in Manuel, two of which already existed in Mafia -- supercold and slime. Again, before Manuel, we said that slimes (in the Slime Tube) were of the "slime" element. This didn't really mean anything though.